src/main/scala/dpla/ingestion3/profiles/ProviderProfiles.scala

```diff
@@ -66,6 +66,16 @@ class OhioProfile extends XmlProfile {
   override def getMapping = new OhioMapping
 }
 
+/**
+  * P2P (Plains2Peaks)
+  * - Colorado and Wyoming
+  */
+class P2PProfile extends JsonProfile {
+  type Mapping = CdlMapping // FIXME Placeholder
```

Comment:
I'd use ??? for this. It has the nice property of blowing up when called.

Reply:
I should format that so it doesn't look like I left a placeholder in my comment: `???` I mean literally three question marks.